### PR TITLE
Add actions

### LINKED ISSUE #17: Replace Jenkins with github actions
We have introduced some [best practices](https://pcic.uvic.ca/confluence/display/CSG/DevOps#DevOps-BestPractices) for our workflow usage. The workflows need to be updated to fall inline with these practices.

### PR DESCRIPTION
**Pull request by nikola-rados**
Resolves #17 

**Changes**
- Add action workflows
- Remove old tools
- Update broken test
- Add badges to `README.md`

### CODE REVIEW

File: .github/workflows/python-ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        pip install -r requirements.txt -r test_requirements.txt
+        pip install .
+    - name: Test with pytest (full)
+      if: github.ref == 'refs/heads/master'
+      run: |
+        py.test -v --tb=short tests
+    - name: Test with pytest (fast)
+      if: github.ref != 'refs/heads/master'
+      run: |
+        py.test -m "not slow" -v --tb=short tests
```
Comment (corviday) [inline .github/workflows/python-ci.yml:33]: I don't think any tests in this repository are marked "slow". Am I missing something?

**Comment by nikola-rados** [comment]
@corviday some of our test suites don't have tests that hit the "slow" threshold. That doesn't mean that this will always be the case though.  To make the CI future-proof (and to avoid the headache it would cause me) I have implemented everything needed to mark tests as slow.

**Comment by nikola-rados** [comment]
Actually, I have missed the `pytest.ini`. I will add that now.

**Comment by corviday** [comment]
Is that documented anywhere? Like, if I add a test that turns out to be really slow, is there some documentation somewhere on marking it or whatever, to use that infrastructure?

**Comment by nikola-rados** [comment]
No there isn't.  I will add it to the [standards](https://pcic.uvic.ca/confluence/display/CSG/DevOps#DevOps-BestPractices).

**Pull request merged**